openbox/event.c

```diff
@@ -988,6 +988,7 @@ static void event_handle_client(ObClient *client, XEvent *e)
             pcon = mouse_button_frame_context(pcon, e->xbutton.button,
                                               e->xbutton.state);
         }
+        break;
```

Comment:
I think this was intentional. IIRC we don't get a button release for the client area since the event gets replayed/redirected to the client. I don't recall why there is a ButtonRelease section then though. How did you test this change? It is a serious behaviour change compared to other things in the PR.

Reply:
I installed it in my computer and tried to send mouse events here and there, everything seemed to work fine.
I tried selecting an area in the desktop, moving windows, resizing them, and general use stuff such as clicking on the UI of Firefox and on websites.
I don't believe that amounts to extensive testing, so any suggestions are welcome.